MarkdownEditor.elm

```diff
@@ -0,0 +1,39 @@
+module MarkdownEditor where
+-- Elm’s "import" keyword works mostly like "require" in node.js.
+-- The “exposing (..)” option specifies that we want to bring the Html module’s contents
+-- into this file’s current namespace, so that instead of writing out
+-- Html.form and Html.label we can just use "form" and "label" without the "Html." prefix.
+import Html exposing(..)
+-- This works the same way; we also want to import the entire
+-- Html.Events module into the current namespace.
+import Html.Events exposing (..)
+-- With this import we are only bringing a few specific functions into our
+-- namespace, specifically "id", "type'", "for", "value", and "class".
+import Html.Attributes exposing (id, type', for, value, class, style)
+-- startapp is boilerplate that most elm apps will use to follow the recommended
+-- elm architecture
+import StartApp.Simple as StartApp
+-- creates a view model that is a navbar
```

Comment:
I don't think this is exactly accurate. `View Model` is a widely used term in MVVM architecture that doesn't apply in this case, and I think it could be confused. Really we `view` which is a function of the `Model` to `Html`. I think this would benefit more from a type annotation (suggested practice is all top level functions have type annotations).

Also, on a related not, how about removing the model entirely? This view does not depend on any model which could double down on the confusion.